CHANGELOG.md

```diff
@@ -4,17 +4,27 @@ Unreleased Changes
 2.1.0 (2019-02-14)
```

Comment:
Make sure also prepare for the changelog entry, you can make it a list of`* Feature -`, versions and date title will be auto bumped later based on `VERSION` file.

Thus, also need to include a update for VERSION file as well : D

Reply:
Done - under the guise of "Upgrading" similar to the 2.0.0 bump.